llvm/include/llvm/SandboxIR/SandboxIR.h

```diff
@@ -1386,6 +1390,27 @@ class PtrToIntInst final : public CastInst {
 #endif // NDEBUG
 };
 
+class BitCastInst : public CastInst {
+public:
+  static Value *create(Value *Src, Type *DestTy, BBIterator WhereIt,
+                       BasicBlock *WhereBB, Context &Ctx,
+                       const Twine &Name = "");
+  static Value *create(Value *Src, Type *DestTy, Instruction *InsertBefore,
+                       Context &Ctx, const Twine &Name = "");
+  static Value *create(Value *Src, Type *DestTy, BasicBlock *InsertAtEnd,
+                       Context &Ctx, const Twine &Name = "");
+
+  static bool classof(const Value *From) {
+    if (auto *I = dyn_cast<Instruction>(From))
+      return I->getOpcode() == Instruction::Opcode::BitCast;
```

Comment:
Please use dyn_cast for PtrToInt "classof" check too.  I wasn't careful to check the use of isa+cast there.

Reply:
I will push an NFC to make this consistent with the rest. Though I still think that the isa + cast pattern looks a bit nicer as it is less verbose.